### DISCUSSION TITLE
Feature Request

### DISCUSSION THREAD

**Discussion by Daarko** [Zrips/CMI#2719] [Enhancement]
**Description of issue or feature request:**
If at all possible, when setting up portals, could we have the option to turn off the particles for people who don't have the correct permission, e.g cmi.command.portal.example

this would allow us to have a quest that opens a portal for someone and actually shows that the portal has been turned on.

**Comment by Va1a** [comment]
Second this, although it's a pretty minor addition, would be pretty cool to have.

**Comment by ghost** [comment]
This was added https://i.imgur.com/IYESCUk.png
